typescript/examples/flatbuffer/output/Grid_generated.ts

```diff
@@ -0,0 +1,7 @@
+// automatically generated by the FlatBuffers compiler, do not modify
```

Comment:
You can use a .gitattributes file to mark these as `linguist-generated` so they don't show up in PRs. See examples in the schemas repo.